### PR TITLE
Upgrade rails to 6.1

### PR DESCRIPTION
**Pull request by sandbergja**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -37,7 +37,7 @@ gem 'jbuilder', '~> 2.0'
 # jquery multiselect plugin for advanced search
 gem 'chosen-rails'
 
-gem 'blacklight-marc', git: 'https://github.com/projectblacklight/blacklight-marc.git', ref: 'c0ff1d9'
```
Comment (sandbergja) [inline Gemfile:40]: This dependency at this particular commit relies on Rails behavior that was removed in rails 6.1

File: spec/helpers/requests/application_helper_spec.rb [hunk 2]
```diff
@@ -94,7 +94,7 @@
...
+      expect(choices).to eq("<div id=\"fields-print__23724990920006421\" class=\"collapse request--print\"><div><ul class=\"service-list\"><li class=\"service-item\">Requests for pick-up typically take 2 business days to process.</li></ul></div><div id=\"fields-print__23724990920006421_card\" class=\"card card-body bg-light\"><input type=\"hidden\" name=\"requestable[][pick_up]\" id=\"requestable__pick_up_23724990920006421\" value=\"{&quot;pick_up&quot;:&quot;PN&quot;,&quot;pick_up_location_code&quot;:&quot;lewis&quot;}\" class=\"single-pick-up-hidden\" autocomplete=\"off\" /><label class=\"single-pick-up\" style=\"\" for=\"requestable__pick_up_23724990920006421\">Pick-up location: Lewis Library</label></div></div>")
```
Comment (sandbergja) [inline spec/helpers/requests/application_helper_spec.rb:97]: [Rails 6.1.5](https://github.com/rails/rails/releases/tag/v6.1.5) added these `autocomplete="false"`.

Comment (christinach) [reply spec/helpers/requests/application_helper_spec.rb:97]: You meant to write in the comment autocomplete="off", correct ?

**Review by christinach — APPROVED** [review]

**Comment by christinach** [comment]
Thanks @sandbergja !

**Pull request merged**